runner.rb

```diff
@@ -1,42 +1,70 @@
-require_relative 'account'
-require_relative 'user'
+require_relative 'bank'
 
-def make_account(name)
-  Account.new({ owner: name })
-end
+# SauceBank runner class
+class Runner
+  attr_accessor :customer
+  def initialize
+    @bank = Bank.new
+    @bank.create_account('latoya')
```

Comment:
As a user, I should be able to enter my name when given a prompt.

Reply:
when you run the program, it does allow you to enter your name, but I wanted to ensure that there was a test account listed, to emulate the requirements given, but any other name, besides `latoya`, would create a new account in the bank.